src/oasis-nexus/api.ts

```diff
@@ -949,3 +955,34 @@ export const useGetConsensusAccounts: typeof generated.useGetConsensusAccounts =
     },
   })
 }
+
+export const useGetConsensusAccountsAddressDelegations: typeof generated.useGetConsensusAccountsAddressDelegations =
+  (network, address, params?, options?) => {
+    const ticker = getTokensForScope({ network, layer: Layer.consensus })[0].ticker
+    return generated.useGetConsensusAccountsAddressDelegations(network, address, params, {
+      ...options,
+      request: {
+        ...options?.request,
+        transformResponse: [
+          ...arrayify(axios.defaults.transformResponse),
+          (data: generated.DelegationList, headers, status) => {
+            if (status !== 200) return data
+            return {
+              ...data,
+              delegations: data.delegations.map(delegation => {
+                return {
+                  ...delegation,
+                  shares: fromBaseUnits(delegation.shares, consensusDecimals),
```

Comment:
Shares don't have decimals like this. I don't know if we want to convert them and use "gigashares" throughout the app

Reply:
For now I will merge as is to avoid conflicts in PR and will address this in a next PR if needed.

Reply:
Notes from discussion:

Out of 
- 213456546455 shares
- 213.456546455 gigashares
- 213.45... B shares
- 213B shares

"213B shares" is probably nicest

We could reuse the code from token supply display https://explorer.dev.oasis.io/mainnet/sapphire/token ; decimals are not very important
![image](https://github.com/oasisprotocol/explorer/assets/3758846/a0f8bf0d-c0d8-4003-a519-4b26e6cb2690)

Reply:
relates to https://github.com/oasisprotocol/explorer/pull/1310

Reply:
Can you verify this in #1310 please?
sample acc: https://0a9444e9.oasis-explorer.pages.dev/mainnet/consensus/address/oasis1qqnk4au603zs94k0d0n7c0hkx8t4p6r87s60axru 
sample validator (with hardcoded GoStaking entity) https://0a9444e9.oasis-explorer.pages.dev/mainnet/consensus/validators/oasis1qpqz2ut6a6prfcjm64xnpnjhsnqny6jqfyav829v/delegators#delegators